core/src/main/c/share/vec_agg_vanilla.cpp

```diff
@@ -185,6 +220,9 @@ int64_t countLong_Vanilla(int64_t *pl, int64_t count) {
 }
 
 int64_t sumLong_Vanilla(int64_t *pl, int64_t count) {
+    if (count == 0) {
```

Comment:
Before it used to return 0, now it's L_MIN, is it tested from SQL?

Reply:
NULL (L_MIN) is consistent with the behavior of other sum functions for other nullable column types. As for the tests, we check for NULL before adding the sum, so this change doesn't affect the SQL result.